Package.swift

```diff
@@ -0,0 +1,81 @@
+// swift-tools-version: 5.6
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "OktaLogger",
+    platforms: [
+        .iOS(.v11),
+        .macOS(.v10_14),
+        .watchOS(.v6)
+    ],
+    products: [
+        .library(
+            /// Complete library with all dependencies
+            name: "OktaLogger",
+            targets: ["OktaLogger"]),
+        .library(
+            /// Core library with no additional dependencies
+            name: "OktaLoggerCore",
+            targets: ["LoggerCore"]),
+        .library(
+            name: "OktaFileLogger",
+            targets: ["FileLogger"]),
+        .library(
+            name: "OktaFirebaseCrashlyticsLogger",
+            targets: ["FirebaseCrashlyticsLogger"]),
+        .library(
+            name: "OktaInstabugLogger",
+            targets: ["InstabugLogger"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/Instabug/Instabug-SP", .upToNextMajor(from: "10.7.0")),
+        .package(url: "https://github.com/firebase/firebase-ios-sdk.git", .upToNextMajor(from: "8.0.0")),
+        .package(url: "https://github.com/CocoaLumberjack/CocoaLumberjack.git", .upToNextMajor(from: "3.6.0"))
+    ],
+    targets: [
+        .target(
+            name: "OktaLogger",
+            dependencies: [
+                .target(name: "FileLogger"),
+                .target(name: "FirebaseCrashlyticsLogger"),
+                .target(name: "InstabugLogger"),
+                .target(name: "LoggerCore")
+            ],
+            exclude: ["AppCenterLogger",
+                      "FileLoggers",
+                      "FirebaseCrashlyticsLogger",
+                      "InstabugLogger",
+                      "LoggerCore",
+                      "Info.plist"]),
+        .target(name: "FileLogger",
+                dependencies: [
+                .product(name: "CocoaLumberjack", package: "CocoaLumberjack"),
```

Comment:
Should it be just `CocoaLumberjackSwift` as `CocoaLumberjackSwift` already includes `CocoaLumberjack`?

Reply:
I was following their advise: `Note that you may need to add both products, CocoaLumberjack and CocoaLumberjackSwift to your target since SPM sometimes fails to detect that CocoaLumerjackSwift depends on CocoaLumberjack.` but just tried it and works ok. 👍